Gruntfile.js

```diff
@@ -82,6 +85,7 @@ module.exports = function(grunt) {
       'test/qunit1.html': 3,
       'test/qunit2.html': 3,
       'http://localhost:9000/test/qunit1.html': 2,
+      'http://localhost:9000/test/qunit3.html?foo=bar&noglobals=': 1,
```

Comment:
This is not valid any more in `QUnit 1.22.0`. Caused by https://github.com/jquery/qunit/commit/a5806f1178b02b9655f19569ac21adf510d8c62f

So the url should be either:
* `http://localhost:9000/test/qunit3.html?foo=bar&noglobals`
* or `http://localhost:9000/test/qunit3.html?foo=bar&noglobals=truthy-value`